index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800&display=swap" rel="stylesheet"/>
+    <link rel="stylesheet" href="./stylesheets/common.css">
+    <link rel="stylesheet" href="./stylesheets/mobile.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="hero">
+
+        <!-- head section -->
+
+        <header>
+            <a href="#" class="header-title">My Logo</a>
+            <nav>
+                <ul class="hamburger">
+                    <li class="drop"><a href="#"></a></li>
+                    <li class="drop"><a href="#"></a></li>
+                    <li class="drop"><a href="#"></a></li>
+                </ul>
+            </nav>
+        </header>
+
+        <!-- Homepage intro section -->
+
+        <main class="hero-intro">
+            <div class="hero-intro-container">
+                <h1>I’m Stanley Glad to see you!</h1>
+                <p>
+                    I’m a software developer! I can help you build a product , feature
+                    or website Look through some of my work and experience! If you
+                    like what you see and have a project you need coded, don’t
+                    hestiate to contact me.
+                </p>
+                <h3>LET’S CONNECT</h3>
+                <ul class="social-icons">
+                    <li><img src="./assets/social/twitter.svg" alt="Twitter"></li>
+                    <li><img src="./assets/social/Linkedin.svg" alt="Linkedin"></li>
+                    <li><img src="./assets/social/medium.svg" alt="Meduim"></li>
+                    <li><img src="./assets/social/github.svg" alt="Github"></li>
+                    <li><img src="./assets/social/angellist.svg" alt="AngelList"></li>
+                </ul>
+            </div>
+        </main>
```

Comment:
- [ ] The previous reviewer requested using a `<section>` tag and this has not been implemented rather you used a `<main>` tag, now while the content of your headline section isn't yet part of the main section but just another section of the header the example below would be a better way of using semantic tags to represent it so please kindly adopt this to your html markup :+1: 
```
        <section class="hero-intro">
            <div class="hero-intro-container">
           </div>
        </section>
```